kernel-crawler/scripts/run-crawler.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+#
+# This is a helper script to add common options to the crawler container
+# for the many invocations in the make file. In particular it supports
+# a log directory. For retrieving the log file (instead of the container
+# using stderr for logging)
+#
+
+def logs_dir():
+    return os.environ.get("CRAWLER_LOGS_DIR", os.getcwd())
+
+
+def run_crawler_container(
+        *args, entrypoint=None, env=None, volumes=None):
+    command = [
+        "docker", "run", "--rm", "-i",
+        "-v", f"{logs_dir()}:/logs",
+        "-e", "CRAWLER_LOGS_DIR=/logs",
+    ]
```

Comment:
We don't need to do this now, but if we are moving to use a python script for driving the crawlers, I would much rather we use something like the [docker SDK](https://docker-py.readthedocs.io/en/stable/) rather than manually creating commands ourselves.